<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Smsender</class>
 <widget class="QDialog" name="Smsender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16000</width>
    <height>16000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smsender</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contactoList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_entry">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>45</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>NÃºmero</string>
   </property>
   <property name="buddy">
    <cstring>number_entry</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>45</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>De</string>
   </property>
   <property name="buddy">
    <cstring>sender_entry</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="sender_entry">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="captcha">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="buddy">
    <cstring>captcha_entry</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="captcha_entry">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>171</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="mensaje_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>45</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>Contactos : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>45</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>202</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>233</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mensaje_text</tabstop>
  <tabstop>number_entry</tabstop>
  <tabstop>sender_entry</tabstop>
  <tabstop>captcha_entry</tabstop>
  <tabstop>send</tabstop>
  <tabstop>add</tabstop>
  <tabstop>edit</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>contactoList</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>Smsender</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
